modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -311,6 +311,14 @@
 		"issueKey": "LPS-196611",
 		"to": "fetchByExternalReferenceCode(param#1#, param#0#)"
 	},
+	{
+		"classNames": [
+			"AssetCategoryLocalService",
+			"AssetCategoryLocalServiceUtil"
```

Comment:
@MicaelleSilvaa, shouldn't we also consider these changes in JSP files?

Reply:
I've checked again and the changes don't apply to JSP